terminus/geometry/point.py

```diff
@@ -23,6 +23,20 @@ def from_tuple(cls, tuple):
         else:
             return cls(tuple[0], tuple[1])
 
+    @classmethod
+    def min(cls, points_list):
```

Comment:
As per the comment above, you can move this to the instance side and have it take a single argument